### PR TITLE
Add Migrating from Stored Procs devblog

### LINKED ISSUE #1414: Migrating from Stored Procedures to dbt Dev Blog
### Contact Details

internal slack

### This issue is being created to promote an approved Discussion.

- [X] I am promoting this issue from an approved Discussion and will link it below.

### Link to the Discussion this Issue is promoting.

Ganz cooridinated with Matt + John to get this prepped

### If you're promoting this from an 'I want to write about...' Discussion, you can copy over the answers to the following questions. Otherwise, please answer these questions below

Already discussed as part of review process

### Link to the initial outline we will develop through this Issue.

https://docs.google.com/document/d/1Lj6Io4EuRlrhFAajD8YEoUVJDkdHxADWFtsZ-9bktA8/edit

### PR DESCRIPTION
**Pull request by johnblust**
Adds content for Matt Winkler's and Natty's migrating from stored procs to dbt blog

### CODE REVIEW

File: website/blog/2022-07-19-migrating-from-stored-procs [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
```
Comment (matt-winkler) [inline website/blog/2022-07-19-migrating-from-stored-procs]: @johnblust it looks like this isn't saved as a markdown file? I'm testing locally and not able to actually load this page.

File: website/blog/2022-07-19-migrating-from-stored-procs [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+title: "Migrating from Stored Procedures to dbt"
+description: "Stored procedures are great, but they eventually become hard to scale. dbt fixes that, but the migration process can seem daunting. Matt Winkler and Natty demystify the process in this blog!"
+slug: change-data-capture-metrics
+
+authors: [matt_winkler, jonathan_natkins]
```
Comment (matt-winkler) [inline website/blog/2022-07-19-migrating-from-stored-procs]: noting also that we don't have a record for natty in `/website/blog/authors.yml`.

**Comment by matt-winkler** [comment]
@jnatkins are you good being included on this, and if so do you want to give a quick bio, linkedin URL and/or headshot photo?

image


cc @johnblust

**Comment by johnblust** [comment]
> @jnatkins are you good being included on this, and if so do you want to give a quick bio, linkedin URL and/or headshot photo?
> 
> image
> 
> cc @johnblust

I got his permission to add that information! Looks like the profile was a bit bugged due to a syntax error, should be good now

**Comment by matt-winkler** [comment]
@johnblust I noticed that this article is fully expanded in the "Featured Posts" section vs. showing a preview like the others. Is that a problem?

Once that's resolved I think we are good here.

**Review by matt-winkler — APPROVED** [review]
Looks good now - thanks @johnblust !!

File: website/blog/2022-07-19-migrating-from-stored-procs (hunk 2 above)
Comment (johnblust) [reply website/blog/2022-07-19-migrating-from-stored-procs]: Good catch, I'll remove that and then merge!

**Pull request merged**